client/src/global/components/schema/Types/Journal.js

```diff
@@ -0,0 +1,108 @@
+import React from "react";
+import PropTypes from "prop-types";
+import lh from "helpers/linkHandler";
+import { renderOffer, renderSeries } from "../helpers";
+import BaseSchema from "../BaseSchema";
+
+export default function Journal({ journal }) {
+  const { attributes, relationships } = journal;
+
+  const renderIssues = () => {
+    const { journalIssues } = relationships;
+
+    if (!journalIssues.length) return {};
+
+    return journalIssues.map(issue => ({
+      "@type": "PublicationIssue",
+      issueNumber: issue.attributes.number,
+      datePublished: issue.attributes.publicationDate
+    }));
+  };
+
+  const renderVolumes = () => {
+    const { journalVolumes } = relationships;
+
+    if (!journalVolumes.length) return {};
+
+    return journalVolumes.map(volume => ({
+      "@type": "PublicationVolume",
+      volumeNumber: volume.attributes.number,
+      datePublished: volume.attributes.publicationDate
+    }));
+  };
+
+  const {
+    slug,
+    title,
+    metadata,
+    publicationDate,
+    createdAt,
+    updatedAt,
+    avatarStyles
+  } = attributes;
+
+  const journalData = {
+    "@type": "Periodical",
+    "@id": metadata.issn,
+    name: title,
+    url: lh.link("frontendJournalDetail", slug),
+    issn: metadata.issn,
+    copyrightHolder: metadata.rightsHolder,
+    copyrightNotice: metadata.rights,
+    dateCreated: createdAt,
+    dateModified: updatedAt,
+    datePublished: publicationDate,
+    publisher: metadata.publisher,
+    hasPart: [...renderIssues(), ...renderVolumes()],
```

Comment:
This is great! One thing I did notice in the documentation is that issues are nested in volumes: https://schema.org/PublicationIssue#eg-0398. If it's not too complicated, I think that would be good to do (at least for issues that do belong to volumes).